appdatabase/migrations/bindata.go

```diff
@@ -118,6 +118,8 @@
 // 1720557751_rename_staging.up.sql (3.258kB)
 // 1720606449_enable_store_confirmation_for_messages_sent.up.sql (191B)
 // doc.go (94B)
+// exit_code_1.txt (2B)
+// test_1.log (56B)
```

Comment:
Oops

Reply:
Yes, these are some go test files that for some reason were picked up by `make generate`.

Reply:
Thanks for pointing this out, removed.